a981d3738014339cd0fae1e7f5aa9429222ee03d..0e6d749a0e18d97fc999372860f37bcc1974b6d0
7a028ca613dbf9193d6c336fb6199431eef763cc..42d017be9282231e3ed49f7536f1f63ef233e216
aa309dcd59e0f41c597b068ad7ffdf866faa6101..210ddb614c64a0f0cca9fda17cd2938facb2237c

From 453980e13015e20dd551531be69b3361b63f22b1 Mon Sep 17 00:00:00 2001
From: Michael Cronenworth <mike@cchtml.com>
Date: Thu, 23 Jan 2020 17:05:21 -0600
Subject: [PATCH] gphoto2.ds: Global variable compatibility update for gcc 10.

Signed-off-by: Michael Cronenworth <mike@cchtml.com>
Signed-off-by: Alexandre Julliard <julliard@winehq.org>
---

 dlls/gphoto2.ds/gphoto2_i.h    | 4 +++-
 dlls/gphoto2.ds/gphoto2_main.c | 2 ++
 2 files changed, 5 insertions(+), 1 deletion(-)


diff --git a/dlls/gphoto2.ds/gphoto2_i.h b/dlls/gphoto2.ds/gphoto2_i.h
index edc4d919fe8..4aad9366b32 100644
--- a/dlls/gphoto2.ds/gphoto2_i.h
+++ b/dlls/gphoto2.ds/gphoto2_i.h
@@ -103,5 +103,7 @@ struct tagActiveDS
     struct jpeg_error_mgr		jerr;
 #endif
-} activeDS DECLSPEC_HIDDEN;
+};
+
+extern struct tagActiveDS activeDS DECLSPEC_HIDDEN;
 
 /* Helper functions */
diff --git a/dlls/gphoto2.ds/gphoto2_main.c b/dlls/gphoto2.ds/gphoto2_main.c
index 437709001a5..6edf82a2503 100644
--- a/dlls/gphoto2.ds/gphoto2_main.c
+++ b/dlls/gphoto2.ds/gphoto2_main.c
@@ -33,4 +33,6 @@
 
 WINE_DEFAULT_DEBUG_CHANNEL(twain);
+ 
+struct tagActiveDS activeDS;
 
 
--
